src/style.css

```diff
@@ -1,3 +1,57 @@
 body {
   margin: 0;
+  background-color: lightslategray;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 501;
+}
+
+.header {
+  display: flex;
+  background-color: white;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 30px;
+}
+
+.navLink {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  color: black;
+  margin-left: 20px;
```

Comment:
you should use margin for `<li>`. All `li` should have `margin-right` besides last one